bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -74,6 +115,16 @@ export async function getDependentStoryFiles(
   const storybookDir = normalize(posix(storybookConfigDir));
   const staticDirs = staticDir.map((dir: string) => normalize(posix(dir)));
 
+  const getPackageName = (modulePath) => {
+    const [, scopedName] = modulePath.match(/\/node_modules\/(@[\w-]+\/[\w-]+)\//) || [];
+    if (scopedName) {
+      return scopedName;
+    }
+
+    const [, unscopedName] = modulePath.match(/\/node_modules\/([\w-]+)\//) || [];
+    return unscopedName;
```

Comment:
One thing to watch out for here is `node_modules/x/node_modules/y` -- we want this to be `y` not `x`. So you'd want to do a negative lookahead for any more `node_modules` I think?